### DISCUSSION TITLE
License restrictions

### DISCUSSION THREAD

**Discussion by nilshoerrmann** [mundschenk-at/php-typography#160] [Ideas]
We are interested in using PHP Typography in a plugin for [Kirby CMS](https://getkirby.com/). From what I understand, the license of this library (GPL2) requires such a plugin to be released under the same license. Although Kirby's source is open it's released under a commercial license which makes me doubt that it can be combined with any plugin under GPL. Same is true for any other CMS under MIT license.

Is this limitation intended?
Would it be possible to open the license using LGPL, MIT or a dual model alongside GPL?

**Comment by mundschenk-at** [comment]
I don't know about the Kirby license, but so long as you use Composer to actually install the library (and not distribute it as part of an aggregated work), you should be fine. 

While I'm open to relicensing under the LGPL, I'm not the sole copyright holder of PHP-Typography. It would therefore be a somewhat prolonged process, depending on how easily @jeffreydking can be contacted nowadays and what his opinion in this regard is.

**Comment by nilshoerrmann** [comment]
Thanks for your quick reply and the information about the shared copyright.

Using Composer would not be our first choice but it would work for us for now. If you and the other copyright holders could agree on a less strict license in the long run that would be very much appreciated! This is a really great library and it would be pleasant to not work in a grey area as GPL can be really tricky.
